pandas/core/arrays/timedeltas.py

```diff
@@ -410,6 +410,22 @@ def std(
             return self._box_func(result)
         return self._from_backing_data(result)
 
+    # ----------------------------------------------------------------
+    # Accumulations
+
+    def _accumulate(self, name: str, *, skipna: bool = True, **kwargs):
+
+        data = self._data.copy()
+
+        if name in {"cumsum", "cumprod"}:
```

Comment:
cumprod shouldn't work

Reply:
This worked before as well, should we open an issue about it? Would like to keep this out of here

Reply:
sure.  can you add a TODO

Reply:
Added